`cargo test -p arrow-cast` fails on main

**Describe the bug**
<!--
A clear and concise description of what the bug is.
-->

Running `cargo test -p arrow-cast` on main fails.

**To Reproduce**
<!--
Steps to reproduce the behavior:
-->

Error:

```sh
failures:

---- arrow-cast/src/display.rs - display::ArrayFormatterFactory (line 292) stdout ----
error[E0432]: unresolved import `arrow_cast::pretty`
  --> arrow-cast/src/display.rs:297:17
   |
 8 | use arrow_cast::pretty::pretty_format_batches_with_options;
   |                 ^^^^^^ could not find `pretty` in `arrow_cast`
   |
note: found an item that was configured out
  --> /Users/jeffrey/Code/arrow-rs/arrow-cast/src/lib.rs:31:9
   |
30 | #[cfg(feature = "prettyprint")]
   |       ----------------------- the item is gated behind the `prettyprint` feature
31 | pub mod pretty;
   |         ^^^^^^

error: aborting due to 1 previous error
```

**Expected behavior**
<!--
A clear and concise description of what you expected to happen.
-->

Should succeed.

**Additional context**
<!--
Add any other context about the problem here.
-->

--- a/arrow-cast/src/display.rs
+++ b/arrow-cast/src/display.rs
@@ -290,6 +290,7 @@ impl<'a> FormatOptions<'a> {
 /// this example requires the `prettyprint` feature.
 ///
 /// ```rust
+/// # #[cfg(feature = "prettyprint")]{
 /// use std::fmt::Write;
 /// use arrow_array::{cast::AsArray, Array, Int32Array};
 /// use arrow_cast::display::{ArrayFormatter, ArrayFormatterFactory, DisplayIndex, FormatOptions, FormatResult};
@@ -351,6 +352,7 @@ impl<'a> FormatOptions<'a> {
 ///        &my_batches,
 ///        &FormatOptions::new().with_formatter_factory(Some(&MyFormatters {}))
 /// );
+/// # }
 /// ```
 pub trait ArrayFormatterFactory: Debug + Send + Sync {
     /// Creates a new [`ArrayFormatter`] for the given [`Array`] and an optional [`Field`]. If the

